tutorials/tut1.rst

```diff
@@ -151,7 +151,7 @@ Go back to jwt.io and change the payload to
 
   {
     "role": "todo_user",
-    "exp": <computed epoch value>
+    "exp": "<computed epoch value>"
```

Comment:
I guess it isn't a valid fix, since the epoch value is supposed to be an integer. If you're interested, I could reword the snippet and its explanation using a dummy integer value instead.

Reply:
I think that would be good. And idea would be to change it to something like `"<computed epoch value> make sure to remove the quotes and make this an integer"`. The least confusing way would be the best. Too bad json doesn't support comments.

Reply:
Please see https://github.com/PostgREST/postgrest-docs/pull/244/commits/eac285d751abe9218d5a1f9ca955dae1924c7ad4